AlphaWallet/AppCoordinator.swift

```diff
@@ -331,7 +331,6 @@ extension AppCoordinator: InitialWalletCreationCoordinatorDelegate {
 extension AppCoordinator: InCoordinatorDelegate {
 
     func didRestart(in coordinator: InCoordinator, reason: RestartReason, wallet: Wallet) {
-        OpenSea.resetInstances()
```

Comment:
Why is this call no longer needed?

Reply:
on wallets switching it resets instances. it was needed when we had had only one active wallet. but for now its non required.